packages/lesswrong/components/common/CommentsListCondensed.tsx

```diff
@@ -16,31 +17,36 @@ const styles = (theme: ThemeType): JssStyles => ({
   },
 });
 
-const LatestPostsDiscussion = ({classes}: {
+const CommentsListCondensed = ({label, contentType, terms, initialLimit, itemsPerPage, showTotal=false, classes}: {
```

Comment:
I made this generic (i.e not subforum related specifically) which I think was _just_ about worth it for the extra complexity